Sources/BraveShared/Extensions/UIActionExtensions.swift

```diff
@@ -14,7 +14,7 @@ extension UIAction {
   ///   1. See a change that happens based on their selection
   ///   2. Ensure that if the view which has shown the context menu were to be removed from the view
   ///      heirarchy as a result of the action, that the context menu does not crash on dismissal.
-  static func deferredActionHandler(_ handler: @escaping UIActionHandler) -> UIActionHandler {
+  public static func deferredActionHandler(_ handler: @escaping UIActionHandler) -> UIActionHandler {
```

Comment:
Lets move this into `BraveUI` instead of `BraveShared`

Reply:
Placed at https://github.com/brave/brave-ios/pull/7589/commits/e5ca33debfb25eca4c069bdc2c2670c82603fb64